app/templates/layout.html

```diff
@@ -99,5 +100,40 @@
         setInterval(refreshContent, 5000); // 5000 milliseconds = 5 seconds
       });
     </script>
+
+    <!-- script for login page quotes-->
```

Comment:
I don't think this script should be added to the layout.html file... would it not be better off putting into the page that's actually going to use it? lets try keep the layout page to consistent elements for ALL pages.

Reply:
Done, adjusted on another commit I did to this branch. Can merge this extend branch to main later.